### PR TITLE
Release/1.3.0

### PR DESCRIPTION
**Pull request by DmitryBespalov**
merge to master

### CODE REVIEW

**Review by sche — APPROVED** [review]

File: MultisigWallet/MultisigWalletDomainModel/Account/AccountUpdateDomainService.swift [hunk 1]
```diff
@@ -54,6 +54,7 @@ open class AccountUpdateDomainService {
     private func updateAccountsBalances(_ accountIDs: [AccountID]) throws {
         try accountIDs.forEach { accountID in
             guard let balance = try self.balance(of: accountID) else { return }
+            print("BALANCE", accountID.tokenID, balance)
```
Comment (sche) [inline MultisigWallet/MultisigWalletDomainModel/Account/AccountUpdateDomainService.swift:57]: please remove or use logger.

Comment (DmitryBespalov) [reply MultisigWallet/MultisigWalletDomainModel/Account/AccountUpdateDomainService.swift:57]: ok

File: MultisigWallet/MultisigWalletImplementations/Repositories/DBTokeListItemRepository.swift [hunk 2]
```diff
@@ -10,22 +10,23 @@ import CommonImplementations
 public class DBTokenListItemRepository: DBEntityRepository<TokenListItem, TokenID>, TokenListItemRepository {
 
     public override var table: TableSchema {
+        // IMPORTANT: If you are adding new field to the SQL table, then add it to the end of the list.
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/Repositories/DBTokeListItemRepository.swift:13]: for DBEntityRepository you can use any order

File: MultisigWallet/MultisigWalletImplementations/Repositories/DBWalletRepository.swift [hunk 3]
```diff
@@ -10,15 +10,20 @@ import CommonImplementations
 public class DBWalletRepository: DBEntityRepository<Wallet, WalletID>, WalletRepository {
 
     public override var table: TableSchema {
+        // IMPORTANT: If you are adding new field to the SQL table, then add it to the end of the list.
```
Comment (sche) [inline MultisigWallet/MultisigWalletImplementations/Repositories/DBWalletRepository.swift:13]: this comment also should be removed

**Pull request merged**